[Fri Jun 30 2017 03:22:43 GMT+0800 (CST)=========================================] /upload
Error: Unexpected field
    at makeError (/home/lex/Workspace/nodejs/server/blog/node_modules/multer/lib/make-error.js:12:13)
    at wrappedFileFilter (/home/lex/Workspace/nodejs/server/blog/node_modules/multer/index.js:40:19)
    at Busboy.<anonymous> (/home/lex/Workspace/nodejs/server/blog/node_modules/multer/lib/make-middleware.js:114:7)
    at emitMany (events.js:146:13)
    at Busboy.emit (events.js:223:7)
    at Busboy.emit (/home/lex/Workspace/nodejs/server/blog/node_modules/busboy/lib/main.js:38:33)
    at PartStream.<anonymous> (/home/lex/Workspace/nodejs/server/blog/node_modules/busboy/lib/types/multipart.js:213:13)
    at emitOne (events.js:115:13)
    at PartStream.emit (events.js:210:7)
    at HeaderParser.<anonymous> (/home/lex/Workspace/nodejs/server/blog/node_modules/dicer/lib/Dicer.js:51:16)
    at emitOne (events.js:115:13)
    at HeaderParser.emit (events.js:210:7)
    at HeaderParser._finish (/home/lex/Workspace/nodejs/server/blog/node_modules/dicer/lib/HeaderParser.js:68:8)
    at SBMH.<anonymous> (/home/lex/Workspace/nodejs/server/blog/node_modules/dicer/lib/HeaderParser.js:40:12)
    at emitMany (events.js:146:13)
    at SBMH.emit (events.js:223:7)
[Fri Jun 30 2017 03:23:32 GMT+0800 (CST)=========================================] /upload
TypeError: Cannot read property 'toLowerCase' of undefined
    at doUpload (/home/lex/Workspace/nodejs/server/blog/routes/upload.js:31:53)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/route.js:127:14)
    at Immediate.<anonymous> (/home/lex/Workspace/nodejs/server/blog/node_modules/multer/lib/make-middleware.js:53:37)
    at runCallback (timers.js:804:20)
    at tryOnImmediate (timers.js:762:5)
    at processImmediate [as _immediateCallback] (timers.js:733:5)
[Sun Jul 09 2017 03:23:52 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 03:24:23 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:29:04 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:32:16 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:33:37 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:35:27 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:39:39 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:40:34 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:46:40 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:47:17 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:48:44 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:49:56 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:51:07 GMT+0800 (CST)=========================================] /
MongoError: topology was destroyed
    at executeCommand (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:291:76)
    at Db.command (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/db.js:336:44)
    at count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2058:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2021:5
    at Promise (<anonymous>)
    at Collection.count (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/collection.js:2020:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:55:25)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:56:12 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:58:05 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 15:59:44 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:02:11 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:03:24 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:04:03 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:04:47 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:07:00 GMT+0800 (CST)=========================================] /
MongoError: Topology was destroyed
    at nextFunction (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:540:25)
    at Cursor.next [as _next] (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:692:3)
    at fetchDocs (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:856:10)
    at toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:883:3)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:840:5
    at Promise (<anonymous>)
    at Cursor.toArray (/home/lex/Workspace/nodejs/server/blog/node_modules/mongodb/lib/cursor.js:839:10)
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:60:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Sun Jul 09 2017 16:13:27 GMT+0800 (CST)=========================================] /tags/MongoDB
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:14:30 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:28:35 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:29:11 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:29:46 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:32:17 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:32:44 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:33:21 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:34:25 GMT+0800 (CST)=========================================] /tags/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:34:35 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:35:34 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:36:07 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:37:20 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:37:31 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:38:00 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:38:38 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 16:39:28 GMT+0800 (CST)=========================================] /tag/test
Error: Not Found
    at app.use (/home/lex/Workspace/nodejs/server/blog/app.js:55:13)
    at Layer.handle [as handle_request] (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:317:13)
    at /home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:335:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/router/index.js:275:10)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:115:13)
    at SendStream.emit (events.js:210:7)
    at SendStream.error (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:282:17)
    at SendStream.onStatError (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:433:12)
    at next (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:743:16)
    at onstat (/home/lex/Workspace/nodejs/server/blog/node_modules/send/index.js:732:14)
    at FSReqWrap.oncomplete (fs.js:152:21)
[Sun Jul 09 2017 17:28:30 GMT+0800 (CST)=========================================] /p/5951f886f8380658f3353db3
TypeError: /home/lex/Workspace/nodejs/server/blog/views/comment.pug:20
    18|           label.col-sm-2.control-label(for='inputWebsite') 网站
    19|           .col.col-sm-8
  > 20|             a.form-control-static #{location.origin}/u/#{user.name}
    21|             input.form-control(name='website', type='hidden', value=`/u/${user.name}`)
    22|           
    23|   if comments && comments.length

Cannot read property 'origin' of undefined
    at eval (eval at wrap (/home/lex/Workspace/nodejs/server/blog/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:344:65)
    at template (eval at wrap (/home/lex/Workspace/nodejs/server/blog/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:438:188)
    at Object.exports.renderFile (/home/lex/Workspace/nodejs/server/blog/node_modules/pug/lib/index.js:428:38)
    at Object.exports.renderFile (/home/lex/Workspace/nodejs/server/blog/node_modules/pug/lib/index.js:418:21)
    at View.exports.__express [as engine] (/home/lex/Workspace/nodejs/server/blog/node_modules/pug/lib/index.js:465:11)
    at View.render (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/view.js:128:8)
    at tryRender (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/application.js:640:10)
    at Function.render (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/application.js:592:3)
    at ServerResponse.render (/home/lex/Workspace/nodejs/server/blog/node_modules/express/lib/response.js:971:7)
    at post (/home/lex/Workspace/nodejs/server/blog/routes/post.js:53:9)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Tue Jul 11 2017 18:02:24 GMT+0800 (CST)=========================================] /
TypeError: Cannot read property 'close' of undefined
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:67:47)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Tue Jul 11 2017 18:02:57 GMT+0800 (CST)=========================================] /
TypeError: Cannot read property 'close' of undefined
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:67:47)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
[Tue Jul 11 2017 18:03:00 GMT+0800 (CST)=========================================] /
TypeError: Cannot read property 'close' of undefined
    at Function.getByPage (/home/lex/Workspace/nodejs/server/blog/models/post.js:67:47)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
